src/components/Avatar.js

```diff
@@ -2,8 +2,16 @@
 // returns an img element with className 'Avatar'
 // img element should have src and alt attributes
 
-export default function Avatar(props) {    
+export default function Avatar(props) {
     const { person } = props;
 
-    // return ...
+    return (
+        <div>
+            <img
+                className="Avatar"
+                src={person.imageUrl}
+                alt="Images"
```

Comment:
We use alt attribute in img element to make them accessible, they should have a meaningful value which represents the image